cmd/tempo-serverless/Makefile

```diff
@@ -0,0 +1,25 @@
+.PHONY: build-docker build-zip test
+
+# builds a docker image named "tempo-serverless" using the pack tool
+#   https://github.com/buildpacks/pack
+build-docker: 
+	go mod vendor
+
+	pack build \
+	  --builder gcr.io/buildpacks/builder:v1 \
+	  --env GOOGLE_RUNTIME=go \
+	  --env GOOGLE_FUNCTION_SIGNATURE_TYPE=http \
+	  --env GOOGLE_FUNCTION_TARGET=Handler \
+	  tempo-serverless
```

Comment:
Something to look into later: according to [their installation docs](https://buildpacks.io/docs/tools/pack/), pack can also be run from a container. It would be nice to replace this with a docker run command, so you don't have to install pack if you don't want to (also nice for CI).

Reply:
I spent some time trying this and was getting permissions errors. Agree that this needs some love before it's integrated into CI.